benchmarks/startup/scenario.py

```diff
@@ -0,0 +1,38 @@
+import os
+import subprocess
+import sys
+
+import bm
+
+
+class Startup(bm.Scenario):
+    telemetry_enabled = bm.var_bool()
+    runtime_metrics_enabled = bm.var_bool()
+    use_ddtrace_run = bm.var_bool()
+    with_trace = bm.var_bool()
+
+    def run(self):
+        code = "import ddtrace"
+        if self.with_trace:
+            # queue and encode trace but don't send to the agent
+            # requests to the agent will raise an exception in a benchmark test
```

Comment:
🤔 this is probably brittle... but... hmm...

Reply:
I updated this benchmark to use httpretty instead of mocking internal methods